### PR TITLE
GET /api/articles/:article_id/comments

### PR DESCRIPTION
**Pull request by chiara2201**

### CODE REVIEW

File: models/models.js [hunk 1]
```diff
@@ -21,3 +21,23 @@ exports.fetchArticleById = (articleId) => {
...
+		)
+		.then((result) => {
+			const fetchedComments = result.rows
+			if (fetchedComments.length === 0) {
+				//resource does not exist
+				return Promise.reject({
+					//forces this into the .catch() in the controller
+					status: 404,
+					message: 'no comments found',
+				})
```
Comment (P-Copley) [inline models/models.js:60]: You're sending a 404 when there are no comments for the article. There being no comments doesn't necessarily mean the article does not exists. 

Consider the situation when the article is first posted. Nobody will have commented at that point so a more appropriate response would be 200 with an empty array of comments. 

As per your other comment you should check if the article_id is in the database for this 👍

**Comment by P-Copley** [comment]
Just the above comment to look at for this one @chiara2201 👍 

Add a test for the empty comments scenario and merge when you're ready 😁
